app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,64 @@
 // Specific CSS for your home-page
+.banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100vh;
+  top: 0;
+
+}
+
+.titles {
+  color: $gold;
+
+  h1 {
+    font-size: 150px;
+    font-weight: bold;
+    font-family: $title-font;
+    text-shadow: 5px 5px 5px rgb(14, 13, 13);
+  }
+
+  h3 {
+    font-size: 30px;
+    color: white;
+    font-family: $body-font;
+  }
+
+  .btn-index {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 50px;
+  }
+
+}
```

Comment:
Why do you need a margin-top ?

There is an empty line in l:32

Reply:
It's just to not be so close to the text above it. Is there a better way to do it?

Reply:
If you add `flex-direction: column;` to your .btn-index then because you justify the content to the end your layout will look like this
![image](https://user-images.githubusercontent.com/75388869/219881483-9ee5cd8a-c18f-43a2-bfb3-1cf5505cd2ab.png)

That will make it more responsive and it should be pretty much similar to what you're trying to do if I understand correctly

Reply:
Oh wait, now that I look at it again your layour is actually this way
![image](https://user-images.githubusercontent.com/75388869/219881645-8b8348e8-dc6f-44de-aae1-5cd88921f68b.png)

Then I understand your reasoning and it is completly fine.
I don't think you need to edit this anymore, beside removing the empty line at l:32